apps/examples/kernel_sample/kernel_sample_main.c

```diff
@@ -468,8 +468,13 @@ static int user_main(int argc, char *argv[])
 	!defined(CONFIG_DISABLE_SIGNALS)
 		printf("\nuser_main: vfork() test\n");
 		vfork_test();
+		check_test_memory_usage();
 #endif
 
+#ifdef CONFIG_SCHED_WORKQUEUE
```

Comment:
in original workqueue Kconfig, there is a optional dependency : SCHED_WORKQUEUE || LIB_USRWORK
please check the dependency